migtests/scripts/live-migration-run-test.sh

```diff
@@ -112,9 +112,8 @@ main() {
 		run_ysql yugabyte "CREATE DATABASE ${TARGET_DB_NAME}"
 	fi
 
-	if [ -x "${TEST_DIR}/add-pk-from-alter-to-create" ]
-	then
-		"${TEST_DIR}/add-pk-from-alter-to-create"
+	if [ "${SOURCE_DB_TYPE}" = "postgresql" ] ; then
```

Comment:
This will use this script for all the cases right?

Reply:
I think you should use `MOVE_PK_FROM_ALTER_TO_CREATE` here as well. to use these only when required

Reply:
Right. My miss. Thanks!